src/panels.cpp

```diff
@@ -1745,9 +1750,9 @@ std::map<std::string, std::vector<window_panel>> initialize_default_panel_layout
 {
     std::map<std::string, std::vector<window_panel>> ret;
 
-    ret.emplace( std::make_pair( "classic", initialize_default_classic_panels() ) );
-    ret.emplace( std::make_pair( "compact", initialize_default_compact_panels() ) );
-    ret.emplace( std::make_pair( "labels", initialize_default_label_panels() ) );
+    ret.emplace( std::make_pair( translate_marker( "classic" ), initialize_default_classic_panels() ) );
```

Comment:
```suggestion
    ret.emplace( std::make_pair( translate_marker( "Сlassic" ), initialize_default_classic_panels() ) );
```

Reply:
I intended to leave it as is. The string is also used as an identifier internally, so I am afraid that changing it would break compatibility with old saves.

Reply:
the worst it could do is revert everyone's configs to default.